OSDOCS-14402:YAML parameter additions (#92896)
openshift/openshift-docs: modules/microshift-ingress-controller-config.adoc

.. Update the {microshift-short} `config.yaml` configuration file by making a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, naming it `config.yaml` and keeping it in the source directory.
* After you create it, the `config.yaml` file takes precedence over built-in settings. The configuration file is read every time the {microshift-short} service starts.

> .. Use a configuration snippet to apply the ingress control settings youmicroshift-ingress-control-concept_{context}

Reviewer: not sure what youmicroshift-ingress-control-concept_{context} is

|`forwardedHeaderPolicy`
|Specifies when and how the ingress controller sets the `Forwarded`, `X-Forwarded-For`, `X-Forwarded-Host`, `X-Forwarded-Port`, `X-Forwarded-Proto`, and `X-Forwarded-Proto-Version` HTTP headers. The following values are valid:

> * `Append`, preserves any existing headers by specifying that the ingressmicroshift-ingress-control-concept_{context} controller appends them.

Reviewer: ```suggestion
* `Append`, preserves any existing headers and appends information in the headers as needed.
```
